README.md

```diff
@@ -71,6 +71,7 @@ If enabled, the CSRF token must be in the payload when modifying data or you wil
 * `options.policy` Object - Object definition of policy.
 * `options.reportOnly` Boolean - Enable report only mode.
 * `options.reportUri` String - URI where to send the report data
+* `options.impl` Function - Optional. Custom implementation to create a report-uri.
```

Comment:
I think it would be better to note in the documentation that the `impl` will have high precedence over the `reportUri`